ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/openshift-release-analysis-prpqr-aggregator-ref.yaml

```diff
@@ -0,0 +1,26 @@
+ref:
+  as: openshift-release-analysis-prpqr-aggregator
+  from_image:
+    namespace: ci
+    name: job-run-aggregator
+    tag: latest
+  commands: openshift-release-analysis-prpqr-aggregator-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  timeout: 6h0m0s
+  grace_period: 30s
+  credentials:
+  - namespace: test-credentials
+    name: crt-job-aggregator-credentials
+    mount_path: /var/run/secrets
+  env:
+  - name: GOOGLE_SA_CREDENTIAL_FILE
+  - name: VERIFICATION_JOB_NAME
+  - name: AGGREGATION_ID
+  - name: EXPLICIT_GCS_PREFIX
+  - name: JOB_START_TIME
+  - name: WORKING_DIR
+  documentation: |-
+    Launches the job-run-aggregator to analyze the release analysis jobs launched with a specific aggregation id.
```

Comment:
Does the document support links? 
NIT: add link to https://github.com/openshift/ci-tools/tree/master/cmd/job-run-aggregator